src/js/view/view.js

```diff
@@ -174,38 +170,17 @@ define([
             }
         }
 
-        function handleMouseDown() {
-            _focusFromClick = true;
-
-            // After a click it no longer has 'tab-focus'
-            _this.trigger(events.JWPLAYER_VIEW_TAB_FOCUS, {
-                hasFocus: false
-            });
+        function handleMouseDown(e) {
+            e.preventDefault();
```

Comment:
This is to prevent the player from being focused with a mouse click - we only want to focus when we are tabbing into the player.